R/bpmnVisualization.R

```diff
@@ -59,13 +59,13 @@ display <- function(bpmnXML, width = NULL, height = NULL, elementId = NULL) {
 #' @name bpmnVisualization-shiny
 #'
 #' @export
-bpmnVisualizationOutput <- function(outputId, width = '100%', height = '400px') {
+bpmn_visualization_output <- function(outputId, width = '100%', height = '400px') {
```

Comment:
⚠️ htmlWidgets expects a function named `<package_name>Output` for the R-binding: http://www.htmlwidgets.org/develop_intro.html#r-binding
This doesn't follow our function naming convention

Reply:
Yes, we can't rename this function.

Reply:
done in f9fe264